plugins/cpp_metrics/service/cxxmetrics.thrift

```diff
@@ -0,0 +1,20 @@
+include "project/common.thrift"
+include "project/project.thrift"
+
+namespace cpp cc.service.cppmetrics
+namespace java cc.service.cppmetrics
+
+enum CppMetricsType
+{
+  ParameterCount = 1,
+  McCabe = 2,
+  LackOfCohesion = 3,
+  LackOfCohesionHS = 4
+}
+
+service CppMetricsService
+{
+  double getCppMetricsForAstNode(
```

Comment:
I am not sure whether it would be efficient or useful in the future to query metrics one by one for AST nodes. Maybe return all metrics for the queried node instead?

Reply:
I left this in the code in case it comes handy some time but I also added a function to collect all available metrics for an AST node.